src/components/Dropdown/Dropdown.spec.tsx

```diff
@@ -0,0 +1,162 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { mount } from "@cypress/react";
+import { Size } from "@utilities/enum";
+import { FC, useState } from "react";
+import Dropdown from "./Dropdown";
+import css from "./Dropdown.module.css";
+import { MenuBlock } from "./DropdownMenu/DropdownMenu";
+import { MENU_ITEM_ACTIVE_ID, MENU_ITEM_TEXT_ID, MENU_ITEM_TITLE_ID } from "./DropdownMenuItem/DropdownMenuItem.spec";
+
+const DROPDOWN_ID = "[data-test-id=dropdown]";
+const DROPDOWN_TRIGGER_ID = "[data-test-id=dropdown-trigger]";
+const DROPDOWN_ITEM_LIST_ID = "[data-test-id=dropdown-item-list]";
+const DROPDOWN_CLEAR_BUTTON_ID = "[data-test-id=dropdown-clear-button]";
+const DROPDOWN_DIVIDER_ID = "[data-test-id=dropdown-divider]";
+const BORDER_STYLE = "1px solid rgb(234, 235, 235)";
+
+const SMALL_ITEMS = [
+    {
+        id: "small-block",
+        menuItems: [
+            {
+                id: "1",
+                title: "Small",
+            },
+            {
+                id: "2",
+                title: "Small second",
+            },
+            {
+                id: "3",
+                title: "Small third",
+            },
+        ],
+    },
+];
+
+const FIRST_ITEM_ID = SMALL_ITEMS[0].menuItems[0].id;
+
+const LARGE_ITEMS = [
+    {
+        id: "large-block-1",
+        menuItems: [
+            {
+                id: "4",
+                title: "Large",
+                subtitle: "Subtitle",
+                size: Size.Large,
+            },
+            {
+                id: "5",
+                title: "Large second",
+                subtitle: "Subtitle",
+                size: Size.Large,
+            },
+        ],
+    },
+    {
+        id: "large-block-2",
+        menuItems: [
+            {
+                id: "6",
+                title: "Large third",
+                subtitle: "Subtitle",
+                size: Size.Large,
+            },
+            {
+                id: "7",
+                title: "Large fourth",
+                subtitle: "Subtitle",
+                size: Size.Large,
+            },
+        ],
+    },
+];
+
+type Props = {
+    menuBlocks: MenuBlock[];
+    size?: Size;
+    placeholder?: string;
+    initialActiveId?: string;
+    clearable?: boolean;
+};
+
+const Component: FC<Props> = ({
+    menuBlocks,
+    size = Size.Small,
+    placeholder = "",
+    initialActiveId = "",
```

Comment:
```suggestion
    placeholder,
    initialActiveId,
```
It can be `undefined`